.travis.yml

```diff
@@ -3,10 +3,11 @@ os:
   - linux
   - osx
 julia:
-  - 0.6
-  - 0.7
   - 1.0
+  - 1.1
 notifications:
   email: false
+before_script:
+  - julia -e 'using Pkg; Pkg.add(PackageSpec(name="MathOptInterface", rev="master"))'
 after_success:
-- julia -e 'cd(Pkg.dir("SCS")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
+  - julia -e 'import Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder())'
```

Comment:
I believe that a simple `- coveralls: true` should do, according to docs:
https://docs.travis-ci.com/user/languages/julia/#coverage

Reply:
It worked on MOI and JuMP but it gives a weird error here:
https://travis-ci.org/JuliaOpt/SCS.jl/jobs/571513843#L936-L978
Maybe it works with codecov but not with coveralls